### PR TITLE
Create PLSduino.yml

### PR DESCRIPTION
**Pull request by makkorjamal**
Partial Least Squares is great for making predictions based on set of features like Predicting weather variability based on Pressure temperature and humidity. This project allows the implementation of this algorithm in Arduino or even an ESP32 Any type of help is appreciated like writing example code, documentation or testing.

### CODE REVIEW

**Comment by makkorjamal** [comment]
I am not sure what caused the initial review fail!

File: _data/projects/PLSduino.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+name: PLSduino
+desc: Partial least squares for Arduino and ESP32
+site: https://github.com/makkorjamal/PLSduino
+tags:
+  - Arduino
```
Comment (shiftkey) [inline _data/projects/PLSduino.yml]:
```suggestion
  - arduino
```

File: _data/projects/PLSduino.yml [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+  - C++
+  - Regression
```
Comment (shiftkey) [inline _data/projects/PLSduino.yml]:
```suggestion
  - regression
```

File: _data/projects/PLSduino.yml [hunk 3]
```diff
@@ -0,0 +1,11 @@
+name: PLSduino
+desc: Partial least squares for Arduino and ESP32
+site: https://github.com/makkorjamal/PLSduino
+tags:
+  - Arduino
+  - C++
```
Comment (shiftkey) [inline _data/projects/PLSduino.yml]:
```suggestion
  - c++
```

File: _data/projects/PLSduino.yml [hunk 4]
```diff
@@ -0,0 +1,11 @@
+name: PLSduino
+desc: Partial least squares for Arduino and ESP32
+site: https://github.com/makkorjamal/PLSduino
+tags:
+  - arduino
+  - c++
+  - regression
+upforgrabs:
+  name: up for grabs
```
Comment (shiftkey) [inline _data/projects/PLSduino.yml]:
```suggestion
  name: upforgrabs
```

**Comment by makkorjamal** [comment]
Sorry for the stupid question.. how can I add tasks?

**Comment by shiftkey** [comment]
> Sorry for the stupid question.. how can I add tasks?

The project only has one open issue, and nothing assigned to that label. This means that it won't be visible after merging because there are no tasks available for contributors:

> A label named [upforgrabs](https://github.com/makkorjamal/PLSduino/labels/upforgrabs) has been found on GitHub but it doesn't contain any open issues. This won't be listed on the site when this is merged but is otherwise fine to proceed.

**Comment by makkorjamal** [comment]
Ah okay.. Thank you for your clarification.. I added the proper label.

**Review by shiftkey — APPROVED** [review]

**Pull request merged**